src/topic_tree_.toit

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+class TopicTreeNode_:
```

Comment:
I'd move this below the main data structure.

Reply:
done.